sphinx/source/quickstart/index.rst

```diff
@@ -0,0 +1,22 @@
+Start Here
+==========
+
+Where to start?
```

Comment:
I don't like this subheading. Can we just remove it entirely, and have `Requirements` and `Building CCF` as links directly under `Start Here` in the left navigation bar?